_docs_integrate/read-attributes-from-peer.md

```diff
@@ -131,7 +131,21 @@ If the Recipient does not want the Sender to read any of its Attributes and, the
+If the Recipient agrees to share a requested Attribute with the Sender, it can accept the associated ReadAttributeRequestItem contained in the Request for reading Attributes. In particular, it must then provide the requested Attribute for its Response to the Request. Depending on whether the Recipient wants to share an Attribute that already exists as a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) or that has to be created first, different parameters for this must be used. These [AcceptReadAttributeRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#acceptreadattributerequestitemparameters) are described in the corresponding section of the Data Model Overview.
+
+Accepting a ReadAttributeRequestItem with a new Attribute or an existing, that wasn't shared with the Sender before, leads to the creation of a LocalAttribute with a [LocalAttributeShareInfo]({% link _docs_integrate/data-model-overview.md %}#localattributeshareinfo) contained within its `shareInfo` property, whose underlying `content` is given by the shared Attribute. An appropriate AcceptResponseItem of type [ReadAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#readattributeacceptresponseitem) is generated, which incorporates the `id` of the LocalAttribute with the LocalAttributeShareInfo in its `attributeId` property and the shared Attribute in its `attribute` property.
+If a new IdentityAttribute is to be shared, a corresponding LocalAttribute without a LocalAttributeShareInfo is additionally created for the Recipient beforehand.
+
+If the ReadAttributeRequestItem is accepted with an existing Attribute that the Recipient already shared with the Sender, an [AttributeAlreadySharedAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#attributealreadysharedacceptresponseitem) will be generated.
```

Comment:
Please stay in the present tense and remove the future tense in this and the following sentences, as the entire scenario page is written in the present tense.

Reply:
You are right! Then please use presence instead of future tense in the sentences you added in which it is grammatically possible (if such exist).